Moving to docsy

ngfreiter opened the pull request:
This moves from the "Learn" theme to "Docsy" and reorganizes the pages accordingly, along with making several stylistic changes and adding some content to the landing page. The documentation itself, which is now under a "docs" tab, is largely untouched.

cmoesel commented on config.toml:
```diff
@@ -1,34 +1,176 @@
+# github_subdir = ""
+
+# Google Custom Search Engine ID. Remove or comment out to disable search.
+gcs_engine_id = "011737558837375720776:fsdu1nryfng"
```
Is this unique to us or are we just using docsy's?  If we're just using docsy's, I assume that's ok?

cmoesel commented on config.toml:
```diff
@@ -1,34 +1,176 @@
+algolia_docsearch = false
+
+# Enable Lunr.js offline search
+offlineSearch = false
```
Did you look at either of the search options above?  Might they be preferable over Google search?

cmoesel commented on content/docs/config.toml:
```diff
@@ -0,0 +1,34 @@
+baseURL = "http://fshschool.org/"
```
Is this whole file a mistake?  It looks like the old "Learn"-based `config.toml` (except with the `theme` set to `"docsy"` instead -- and it's in the wrong place...

cmoesel commented on content/_index.html:
```diff
@@ -0,0 +1,66 @@
+
+
+{{% blocks/feature icon="fas fa-fire" title="State of the art" %}}
+The FSH Specification evoles as FHIR does, ensuring you have access to the latest and greatest features of FHIR. 
```
Might some people think that this means FSH should work w/ not-yet-released FHIR versions?  For example, FHIR is currently evolving toward R5 (and has been for about a year now).  If FSH were really evolving w/ FHIR, wouldn't we support the currently evolving R5?  Or am I reading too much into this?

ngfreiter replied:
I don't think you're reading too much into this, but a lot of this text on the front page was made under the assumption that you and @markkramerus may want to make edits to the content itself, so don't feel like having a "State of the art" section is necessarily required. You're free to ask for different sections altogether.

cmoesel commented on content/_index.html:
```diff
@@ -0,0 +1,66 @@
+{{% /blocks/feature %}}
+{{< /blocks/section >}}
+
+{{< blocks/section >}}
```
Should we get permission from these guys before putting their quotes on our web site?

ngfreiter replied:
These quotes are taken from the "brag sheet" that @markkramerus had, so we had permission (I assume) to use those quotes there. I can ask again if you think this context is different enough to warrant it.

cmoesel replied:
I think @markkramerus handled it for the brag sheet, so I'll let him decide if he thinks we need to request permission or give a heads up on the use of these quotes here.

cmoesel commented on layouts/partials/community_links.html:
```diff
@@ -0,0 +1,28 @@
+</div>
+<div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
+<h2>Develop and Contribute</h2>
+<p>If you want to get more involved by contributing to FSH, join us here:
```
The links point to GH for SUSHI, not FSH -- so it seems a little inaccurate to call it "contributing to FSH".  Also, I think we should have links for contributing to this site (if people want to enhance docs, provide more tutorials, etc), the playground, and eventually GoFSH.  But at least SUSHI and  _this site_ for now.

cmoesel commented on layouts/partials/page-meta-links.html:
```diff
@@ -0,0 +1,28 @@
+{{ $issuesURL := printf "%s/issues/new?title=%s" $gh_repo (htmlEscape $.Title )}}
+{{ if $gh_project_repo }}
+{{ $project_issueURL := printf "%s/issues/new" $gh_project_repo }}
+<a href="{{ $project_issueURL }}" target="_blank"><i class="fab fa-github fa-fw"></i> {{ T "post_create_project_issue" }}</a>
```
I'm not sure we need a link to SUSHI issues in the sidebar -- especially since we have one in the footer.  And the test "Create a project issue" is confusing... which project?

I think it would be better to have a link that said "Report a documentation issue" or something like that which goes to GitHub issues for this site.

ngfreiter replied:
I kinda like having the link to SUSHI bugs, since most of this documentation does pertain to SUSHI, and I know personally I've never looked too close at footer links. So I'd like to keep that unless you really dislike it. But I agree that the name should be changed to reference SUSHI, and I can add a "report a documentation issue" link as well.

cmoesel replied:
Sounds like a reasonable compromise.

ngfreiter commented:
> FSH Playground isn't ready for prime time yet. Should we remove references to it for now or are we going to hold back all of FSH School until it is ready?

I'm not sure. It depends on what you consider ready for prime time. FSHPlayground (or FSHOnline) is functional.

> Also... I noticed on some pages we get partial shorties (e.g., I see just the school master's hat in the "About" page). I thought you were going to use the background without shorties for those pages?

Yeah it is just the about page. I need a copy of the background without Shorty. @markkramerus can you share that with me?

> I think maybe we should add this to the README, as it's not obvious why `hugo server` is failing if you're not aware that we use submodules (or not aware how submodules work).
Will do

cmoesel commented:
> > FSH Playground isn't ready for prime time yet. Should we remove references to it for now or are we going to hold back all of FSH School until it is ready?
> 
> I'm not sure. It depends on what you consider ready for prime time. FSHPlayground (or FSHOnline) has some functionality.

I think it needs the console before we really call it functional, else it doesn't tell users when something is wrong or what specifically is wrong.  And based on where @markkramerus set the bar on calling FSH School ready, I'm guessing we'll need to style the playground some more before he wants to put it front and center.

ngfreiter commented:
>I think it needs the console before we really call it functional, else it doesn't tell users when something is wrong or what specifically is wrong. And based on where @markkramerus set the bar on calling FSH School ready, I'm guessing we'll need to style the playground some more before he wants to put it front and center.

>Also, we should move it from github.com/standardhealth to github.com/FSHSchool -- but that should be pretty trivial.

Yeah that's fair. Do you want me to make it a "Coming soon!" like the US Core Patient Tutorial is right now, or just hide any reference to it for now?

ngfreiter commented on config.toml:
```diff
@@ -1,34 +1,176 @@
+
+# A link to latest version of the docs. Used in the "version-banner" partial to
+# point people to the main doc site.
+url_latest_version = "https://example.com"
```
This was made using the example site from docsy, I will clean up config.toml though.

cmoesel commented:
We can let @markkramerus decide how he wants to handle the rollout of FSH Online.  Mark?  Do you want us to:
(a) keep all references to FSH Online and link to FSH Online as it is whenever we release this site?
(b) keep all references to FSH Online but just mark it as "Coming Soon" w/ no links
(c) temporarily remove all references to FSH Online

ngfreiter commented:
https://github.com/FSHSchool/site/pull/1/commits/4c9367bff596e36e2083de32709cf78f27eeb883 should contain all of the requested changes, except those which are waiting on a response.

ngfreiter commented:
Ok @cmoesel, https://github.com/FSHSchool/site/pull/1/commits/ff97124f7212f64cf711be366446989c12d82c5c should resolve all outstanding comments. I added a blank background to the about page, and added a coming soon for when people click on a link leading to FSH Online.
